models/order/src/parcel/transformers.ts

```diff
@@ -0,0 +1,19 @@
+import { Transformer } from '@commercetools-test-data/core';
+import type { TParcel, TParcelGraphql } from './types';
+
+const transformers = {
+  default: Transformer<TParcel, TParcel>('default', {
+    buildFields: ['items', 'custom', 'measurements', 'trackingData'],
```

Comment:
Since we're using the same list in all transformers, perhaps we can extract it to a constant.